### PR TITLE
feat: add g-md-editor classname to editor dom elem, change default prefix for internal classnames

### PR DESCRIPTION
**Pull request by d3m1d0v**
- changed editor's classname `yfm-editor` -> `g-md-editor`
- added `yfm-editor` classname to editor in `YfmDist` extension
- changed prefix of class name (`ye-` -> `g-md-`) when creating class names using helper from `src/classname`
- rename css variable: `--ye-toolbar-sticky-offset` ––> `--g-md-toolbar-sticky-offset`
- rename math css variables:
  - `--ye-math-back-color` ––> `--g-md-math-back-color`
  - `--ye-math-hover-color` ––> `--g-md-math-hover-color`

### CODE REVIEW

File: src/bundle/MarkupEditorView.scss [hunk 1]
```diff
@@ -3,8 +3,8 @@
 $selection-color: rgba(98, 146, 255, 0.2);
 $toolbar-height: 28px;
 
-.ye-markup-editor,
-.ye-markup-preview {
+.g-md-markup-editor,
```
Comment (makhnatkin) [inline src/bundle/MarkupEditorView.scss:6]: question (not blocks):

What is the reasoning behind choosing a class name and what does -g stand for? Is this consistent with the Gravity UI?

File: src/extensions/yfm/YfmCut/nodeviews/yfm-cut-title.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
-.yfm-cut-title .ye-yfm-cut-title-inner {
+.yfm-cut-title .g-md-yfm-cut-title-inner {
```
Comment (makhnatkin) [inline src/extensions/yfm/YfmCut/nodeviews/yfm-cut-title.scss:1]: question (not blocks):
.yfm-cut-title  is is correct?

File: src/extensions/yfm/YfmDist/index.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const YfmDist: ExtensionAuto = (builder) => {
             new Plugin({
                 props: {
                     attributes: {
-                        class: 'yfm yfm_no-list-reset',
+                        class: 'yfm yfm_no-list-reset yfm-editor',
```
Comment (makhnatkin) [inline src/extensions/yfm/YfmDist/index.ts:20]: why here yfm-editor?

Comment (d3m1d0v) [reply src/extensions/yfm/YfmDist/index.ts:20]: for backward compatibility, as some services use this class name to override styles in markdown content in editor

File: src/extensions/yfm/YfmCut/nodeviews/yfm-cut-title.scss (hunk 2 above)
Comment (d3m1d0v) [reply src/extensions/yfm/YfmCut/nodeviews/yfm-cut-title.scss:1]: yes, this css selector is correct

File: src/bundle/MarkupEditorView.scss (hunk 1 above)
Comment (d3m1d0v) [reply src/bundle/MarkupEditorView.scss:6]: yes, `g-md-` its short for Gravity Markdown Editor

**Review by makhnatkin — APPROVED** [review]

**Pull request merged**